rudder-core/src/main/scala/com/normation/rudder/db/Doobie.scala

```diff
@@ -56,6 +56,7 @@ import com.normation.rudder.services.reports.RunAndConfigInfo
 import org.slf4j.LoggerFactory
 import doobie.util.log.ExecFailure
 import doobie.util.log.ProcessingFailure
+import scala.language.implicitConversions
```

Comment:
this seems unused

Reply:
it's necessary for using the keyword without warning with -Xlint compiler flag